package.json

```diff
@@ -10,7 +10,9 @@
     "lint": "eslint .",
     "test": "npm run lint && npm run run-tests",
     "run-tests": "lab bin/ server/ lib/ -t 80 -l -e 'development' -P spec",
-    "test-cov-html": "lab bin/ server/ lib/ -r html -o coverage.html -P spec"
+    "test-cov-html": "lab bin/ server/ lib/ -r html -o coverage.html -P spec",
+    "webpack-watch": "webpack",
```

Comment:
You shouldn't reference the tool you're using in the task name.

I think it will be better to use something like:
```
"dev": "webpack",
"deploy": "webpack --deploy",
```